### DISCUSSION TITLE
How to push live stream for youtube?

### DISCUSSION THREAD

**Discussion by SignedQiu** [scivision/PyLivestream#88] [Q&A]
I follewed the instructions and pushed live stream for Twitch and Bilibili successfully, but when I push the live stream for Youtube, it failed. My configurations are below(I have reset the stream key):
![image](https://user-images.githubusercontent.com/66827919/180222590-323a1683-699d-4cb4-99d9-dbfaf96b8947.png)
![image](https://user-images.githubusercontent.com/66827919/180222661-3ff8001a-334a-48d7-b7cd-99e6cfc55fe2.png)
 Errors below:
![image](https://user-images.githubusercontent.com/66827919/180222992-2d29e1ac-9538-4fc4-ab92-02b7abd51f54.png)

**Comment by scivision** [comment]
I'm not sure, the command line output for ffmpeg looks correct. Did you initiate the livestream first in the YouTube Studio webpage?
